lnst/Controller/Namespace.py

```diff
@@ -265,3 +265,9 @@ def _update_device_id(self, value, name):
             return
 
         dev._id = name
+
+    def keep_addrs_on_dev_down(self, device: Device) -> None:
+        self.run(f"echo 1 > /proc/sys/net/ipv6/conf/{device.name}/keep_addr_on_down")
```

Comment:
this sounds like it should be a method for the `Device` class instead of in the `Namespace` class...